src/common/options/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
+# A custom target using comon_options_srcs will generate YAML files and its
+# corresponding source files and header files. Remove the headers target to
+# prevent headers from generating twice.
```

Comment:
i don't get this. if headers are already generated and updated, why they are generated twice?